splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -36,8 +36,17 @@
 public class HttpService {
     // For debugging purposes
     private static final boolean VERBOSE_REQUESTS = false;
-    public static boolean useTLS=false;
+    public static boolean useTLS = false;

```

Comment:
I'm going to suggest removing this flag, we introduced this fairly recently (5 months ago) and this should already be possible with `HttpService.setSslSecurityProtocol(SSLSecurityProtocol.TLSv1_2)` and also we are not properly handling the case when this is changed - the sslSocketFactory also needs to be updated when this flag is changed (it is not currently).